.github/workflows/release.yaml

```diff
@@ -1,17 +1,31 @@
 name: "Release"
 on:
-  push:
-    tags: [ 'v*' ]
-
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
 jobs:
   release:
+    if: |
+        github.event.pull_request.merged == true && github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.pull_request.user.login == 'github-actions[bot]' && github.event.pull_request.user.type == 'Bot'
```

Comment:
Can we also check if the source branch of the PR starts with `releases/`? Maybe we use this bot account for more things in the future, and this workflow should only run on releases.
```
startsWith(github.event.pull_request.head.ref, 'releases/')
```

Reply:
Done!